src/test/resources/application.yml

```diff
@@ -6,8 +6,6 @@ spring:
     hikari:
       maximum-pool-size: 2
 
-maxTimeSeriesCorrectedData: 10
-
```

Comment:
Same here, do we need the `-` on line 10?

Reply:
Update, I'm confusing myself, this is a diff -, not actual characters being removed.  Ignore!